README.md

```diff
@@ -64,6 +64,8 @@ package.
 Check your package manager on how to install these dependencies (e.g.,
 `apt-get install libdw-dev libelf-dev` in Debian-based systems). Note that you may need to tell the
 compiler where to find the header and library files of the dependencies for the build to succeed.
+This is handled at Pystack level provided the users install pkg-config (e.g.,
+`apt-get install pkg-config` in Debian-based systems).
```

Comment:
Let's rephrase this a little bit:
```suggestion
If `pkg-config` is available (e.g. `apt-get install pkg-config` on Debian-based systems), it will
automatically be used to locate the libraries and configure the correct build flags.
```